### PR TITLE
[operator-opamp-bridge][chore]: Refactor OpAMPBridge Webhook

### PR DESCRIPTION
**Pull request by avadhut123pisal**
**Description:** 
This PR updates the OpAMPBridge Webhook code to make it consistent with the changes that are made in Collector webhook as a part of #2210. 

**Link to tracking Issue:** NA

**Testing:** Tested locally. 

**Documentation:** NA

### CODE REVIEW

File: config/webhook/manifests.yaml [hunk 1]
```diff
@@ -137,7 +137,7 @@ webhooks:
       namespace: system
       path: /validate-opentelemetry-io-v1alpha1-opampbridge
   failurePolicy: Fail
-  name: vopampbridge.kb.io
+  name: vopampbridgecreateupdate.kb.io
```
Comment (jaronoff97) [inline config/webhook/manifests.yaml:140]: why did this name change?

Comment (avadhut123pisal) [reply config/webhook/manifests.yaml:140]: https://github.com/avadhut123pisal/opentelemetry-operator/blob/c3088a72c7bd0af9f7d3283875cfdb860cc309f2/apis/v1alpha1/opampbridge_webhook.go#L37C1-L37C1

The change has been done considering the validation webhook configurations of Collector resource. 
One webhook is for create & update operations (with failure policy as Fail) and another one is for delete operation (with failure policy as Ignore). Previously, by default there was a single webhook for all the operations.

File: apis/v1alpha1/opampbridge_webhook.go [hunk 2]
```diff
@@ -15,33 +15,69 @@
...
 
 import (
+	"context"
 	"fmt"
 	"strings"
 
+	"github.com/go-logr/logr"
+
+	"github.com/open-telemetry/opentelemetry-operator/internal/config"
+
```
Comment (jaronoff97) [inline apis/v1alpha1/opampbridge_webhook.go:25]: the structure of these imports is wrong, though idk why our linter isn't catching it...

**Review by jaronoff97 — APPROVED** [review]

File: apis/v1alpha1/opampbridge_webhook.go (hunk 2 above)
Comment (jaronoff97) [reply apis/v1alpha1/opampbridge_webhook.go:25]: im going to leave this in for now, and then we can figure out why the linter isn't catching this in a follow up

Comment (avadhut123pisal) [reply apis/v1alpha1/opampbridge_webhook.go:25]: > the structure of these imports is wrong, though idk why our linter isn't catching it...

Yeah. Sometimes, it throws error for correct import structure too.

**Pull request merged**